README.md

```diff
@@ -148,8 +148,22 @@ func main() {
 		fmt.Println("Successfully created a new Client object!")
 	}
 
+	indexName := "my-serverless-index"
+
+	indexes, err := pc.ListIndexes(ctx)
+	if err != nil {
+		log.Fatalf("Failed to list indexes: %v", err)
+	}
+
+	for _, idx := range indexes {
```

Comment:
The returned error `err` for `CreateServerlessIndex` currently has a message for when the index already exists, so I don't think this is necessary. The "bug" was in the if-statement after this. If the index fails to create, it should log `indexName` not `idx.Name` since idx has not been created.

Reply:
Yeah, look on n 174, we can't access `idx` if there was an error. Nice catch, Evan.

Reply:
Ahh thank you! I was misunderstanding your feedback from the bug bash :)